stories/setup/main.js

```diff
@@ -0,0 +1,30 @@
+require('@babel/register');
+
+const webpackProdConfig = require('webpack.prod.config.babel').default;
+const { getPlugins, getStyleRules } = require('../../webpack-common');
+
+module.exports = {
+  stories: ['../index.js'],
+  addons: ['storybook-addon-rtl/register', '@storybook/addon-docs'],
+  webpackFinal: (config) => {
+    return {
+      ...config,
+      node: webpackProdConfig.node,
+      optimization: {
+        minimizer: [],
+      },
+      module: {
+        ...config.module,
+        rules: [
+          ...config.module.rules,
+          ...getStyleRules({ bundleStylesWithJs: true }),
+        ],
+      },
+      plugins: [
+        ...config.plugins,
+        ...getPlugins({ includeLoadablePlugin: false }),
+      ],
+      resolve: webpackProdConfig.resolve,
```

Comment:
this is more or less what we had in the webpack config files before but now storybook wants us to declare the config here.